rct229/data/ashrae_90_1_table_G3_1_3_7.json

```diff
@@ -0,0 +1,80 @@
+{
+    "heat_rejection_leaving_water_temp":[
+        {
+            "climate_zone": "CZ0A",
```

Comment:
I suggest dropping the "CZ" prefix for the climate zones.  Table G3.1.3.11 does not have the "CZ" nor is it used in other JSON tables.

Reply:
"CZ" prefix is from the [schema](https://github.com/open229/ruleset-model-description-schema/blob/master/docs229/Enumerations2019ASHRAE901.schema.json#L872-L893).

The reason I added CZ in the Json was trying to avoid to create a map in the table 3.1.3.11 function.
It seems the table 3.2 already have a similar map there. I don't have strong feeling about one or another, but just want to know your preference?